src/main/java/gate/corpora/DocumentStaxUtils.java

```diff
@@ -1004,7 +1003,7 @@ public static void writeDocument(Document doc, OutputStream outputStream, String
    *          will be treated as the default set. All other entries are
    *          treated as named annotation sets.
    * @param xsw the StAX XMLStreamWriter to use for output
-   * @throws GateException if an error occurs during writing
+   * @throws XMLStreamException if an error occurs during writing
```

Comment:
This might be something we need to handle in extractor.. new exception type. If it is not wrapped in ExecutionExeception.

Reply:
Added to follow up list: https://jira.corp.docusign.com/confu/display/ML/POC%3A+Update+Gate